flybywiresim/aircraft
# fix(apu/gpws/fms): stop using sim altimeters

Fixes #[issue_no]

## Summary of Changes

I was going to finally do the work to sort the altitude/baro correction stuff out but I saw it would conflict with https://github.com/flybywiresim/aircraft/pull/7587. Might as well still bring these fixes that I had already done (working on getting rid of all usages of the sim altimeter vars).
- Switch the APU to using inlet pressure for it's altitude-related limits (as in the real unit).
- Switch the EGPWS to using the ADR1 baro correct altitude from the captain's side, as in the real one.
- Switch the GPS MONITOR page to using WGS84 GPS altitude (it should be corrected to EGM96 but that's non-trivial and hardly noticable for the most part, will do that later in the MMR).

No user visible changes so no changelog.

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions

- Check that the "hundred above" and "minimums" calls work.
- Check that the GPS MONITOR page displays GPS ALT.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
